src/app/components/Curation/CurationGrid/index.styles.tsx

```diff
@@ -13,10 +13,6 @@ const styles = {
             width: '100%',
             paddingInlineStart: 0,
           },
-
-          '[class*="ChildWrapper" i]': {
```

Comment:
What does `i` mean in this context? And do we have a like-for-like replacement with our new CSS?

Reply:
Something to do with case sensitivity, in the new case it doesn't apply anymore as we're the element targeting directly https://www.w3.org/TR/selectors-4/#attribute-case